h/templates/embed.js.jinja2

```diff
@@ -1,3 +1,17 @@
+{% if client_boot_script %}
+(function () {
+  var configEl = document.createElement('script');
+  configEl.className = 'js-hypothesis-settings';
+  configEl.type = 'application/json';
+  configEl.textContent = JSON.stringify({
+    assetRoot: '{{ client_asset_root }}',
+    sidebarAppUrl: '{{ app_html_url }}',
```

Comment:
The defaults are now hardcoded to correct values in the client, with one caveat - the `/app.html` URL can only be hardcoded to one value - so if we remove this config it won't be possible to load the QA version of `/app.html` on the QA version of the service.